README.md

```diff
@@ -1,32 +1,32 @@
 # xmtp-android
 
-![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-yellow)
+![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Beta-yellow)
```

Comment:
I honestly think we could move this to GA in this commit as well. I think it's ready.

Reply:
O I guess thats what Beta is 😸

Reply:
Ah, I love the idea of moving it to GA - TY - let's do it! After the renaming, GA = Production - so let me make that update.

Another requirement for GA is publishing the SDK reference docs. Thanks to you, the dokkaHtml docs are generating - I think we just wanted to find a way to publish them to a GitHub Page and then autoupdate whenever there is a merge to `main`?

Reply:
I updated the status to Production. I think I can merge this PR for now -- and we can work on getting the SDK reference docs up and linked in the README before we make a more formal announcement. WDYT, @nplasterer?

Reply:
I think that sounds great! Lets merge this.